### PR TITLE
display a warning message if part[1] is missing mention of the id key…

### PR DESCRIPTION
**Pull request by RutviShukla**
… related to [race] or [era]

Hey, I made a set of if/else statements to help resolve this issue related to a warning message that needed to be implemented for this section.

### CODE REVIEW

File: src/help/help_impl.cpp [hunk 1]
```diff
@@ -337,6 +337,16 @@ std::vector<topic> generate_topics(const bool sort_generated,const std::string &
 		} else {
 			WRN_HP << "Found a topic generator that I didn't recognize: " << generator << "\n";
 		}
+
+		if(parts.size()>1 && part[1] == "era"){
```
Comment (stevecotton) [inline src/help/help_impl.cpp:341]: I think `part[1]` is a typo for `parts[1]`.

**Comment by Pentarctagon** [comment]
Is there an issue here on github that this resolving?

File: src/help/help_impl.cpp (hunk 1 above)
Comment (RutviShukla) [reply src/help/help_impl.cpp]: Fixed the typo!

Comment (RutviShukla) [reply src/help/help_impl.cpp]: @Pentarctagon trying to contribute to resolving issue 5036

Comment (Pentarctagon) [reply src/help/help_impl.cpp]: Alright, in that case, if you put text in the commit message like "Fixes #5036", then github will automatically close that issue when that commit is merged.

**Comment by Pentarctagon** [comment]
It looks like this doesn't build, based on the CI output.

File: src/help/help_impl.cpp (hunk 1 above)
Comment (RutviShukla) [reply src/help/help_impl.cpp]: @Pentarctagon Thank you! I plan on working on it a bit more to possibly improve it.

**Comment by Pentarctagon** [comment]
@RutviShukla Are you building this yourself locally?

**Comment by cooljeanius** [comment]
> It looks like this doesn't build, based on the CI output.

Do you remember what the CI output was? It appears to have disappeared...

**Comment by Pentarctagon** [comment]
I don't.

**Pull request closed without merging**